### ISSUE TITLE
Source/Destination Path behaviour not the same as other DSC resources

### ISSUE THREAD

**Issue by Zuldan** [mgreenegit/xRobocopy#10]
When a path has spaces in it you need to quote the path. I understand this is a limitation of Robocopy (as it's accepting text based input) but the resource interaction should be standard. The user should be allowed to specify the path without quotes (like Archive and File resource) and the quotes should be handled within the xRobocopy resource.

Example below; Forced to use quotes when spaces exist in destination path.

```
    xRobocopy CopyMyFiles        {
        SubdirectoriesIncludingEmpty = $True
        Source                       = "\\myserver\mymodules"
        Destination                  = "`"C:\Program Files\WindowsPowerShell\Modules`""
    }
```

**Comment by Arturas-K** [comment]
@Zuldan @KarolKaczmarek this can be closed now, in #16 PR command are passed as array, so now path behavior should be consistent.

**Comment by Zuldan** [comment]
Thank you @Arturas-K  for getting this fixed.
